src/cr/cube/cubepart.py

```diff
@@ -561,7 +609,9 @@ def columns_scale_median_margin(self):
     @lazyproperty
     def counts(self):
         """2D np.float64 ndarray of weighted cube counts."""
-        return self._assembler.weighted_counts
+        return self._assemble_matrix(self._measures.weighted_counts.blocks)
+
+    weighted_counts = counts
```

Comment:
I don't know python evaluation well enough to know: Does this remove the "lazyness"? so that counts are always calculated because you assing weighted_counts = counts?

If so do we care enough?

Reply:
it does not remove the lazyness; the weighted_counts attribute is just a new name for the same property getter